media-libs/phonon/phonon-9999.ebuild

```diff
@@ -44,10 +44,10 @@ RDEPEND="
 		designer? ( dev-qt/designer:5 )
 	)
 	pulseaudio? (
-		dev-libs/glib:2
-		>=media-sound/pulseaudio-0.9.21[glib]
+		dev-libs/glib:2[${MULTILIB_USEDEP}]
+		>=media-sound/pulseaudio-0.9.21[glib,${MULTILIB_USEDEP}]
 	)
-	zeitgeist? ( dev-libs/libqzeitgeist )
+	zeitgeist? ( dev-libs/libqzeitgeist[${MULTILIB_USEDEP}] )
```

Comment:
Can zeitgeist support just block multilib instead?  That use flag is masked for this package on amd64 anyways, and other arches just block it.  It looks like converting that will be a lot of work for not so much benefit.
If unmasking the useflag is wanted, I can see about working on its dependencies.  Would that work?